### ISSUE TITLE
[java] UnusedImports: False positive if adding wildcard imports and using only static fields

### ISSUE THREAD

**Issue by agathver** [pmd/pmd#2205]
**Affects PMD Version:** 
6.17

**Rule:**
[UnusedImports]https://pmd.github.io/latest/pmd_rules_java_bestpractices.html#unusedimports

**Description:**

PMD flags a false positive when:

1. Import a package with wildcard
2. Use only static final constants from the one or more classes from the package

**Code Sample demonstrating the issue:**

```java
package com.agathver;

import java.nio.charset.*;

public class App {
    public static void main(String[] args) {
        System.out.println(StandardCharsets.UTF_8.toString());
    }
}
```

**Running PMD through:** *Gradle*

**Gradle, JDK and OS information:**

```
------------------------------------------------------------
Gradle 6.0.1
------------------------------------------------------------

Build time:   2019-11-18 20:25:01 UTC
Revision:     fad121066a68c4701acd362daf4287a7c309a0f5

Kotlin:       1.3.50
Groovy:       2.5.8
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          11.0.5 (AdoptOpenJDK 11.0.5+10)
OS:           Mac OS X 10.15.2 x86_64
```

This is very similar to #2016

**Comment by agathver** [comment]
It appears that it's fixed in 6.20, and somehow our pmd version was getting locked to 6.15
